theano/sandbox/linalg/ops.py

```diff
@@ -280,8 +244,6 @@ def transinv_to_invtrans(node):
 @register_stabilize
 @local_optimizer([Dot, Dot22])
 def inv_as_solve(node):
-    if not imported_scipy:
```

Comment:
Yes it did.

I propose importing `imported_scipy` from `theano.tensor.slinalg` as a fix?